src/Polly.Specs/GlobalUsings.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Global using directives
```

Comment:
IMHO using the `<Using Include="..." />` item syntax in the project file is nicer that having to create an extra `.cs` file.

You can also put them in the `Directory.Build.props` file and cascade them to all projects and use conditions to activate them (`IsTestProject == true`) etc.

Reply:
`<Using Include="..." />`. ok cool. i have gone that approach in some projects. I will move this PR over

Reply:
```
You can also put them in the Directory.Build.props file and cascade them to all projects and use conditions to activate them (IsTestProject == true) etc.
```

will do